xapian-letor/include/xapian-letor.h

```diff
@@ -0,0 +1,39 @@
+/** @file xapian-letor.h
+ *  @brief Public interfaces for the Xapian library.
+ */
+// Copyright (C) 2003,2004,2005,2007,2008,2009,2010,2012,2013,2015,2016 Olly Betts
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+
+#ifndef XAPIAN_INCLUDED_XAPIAN_LETOR_H
+#define XAPIAN_INCLUDED_XAPIAN_LETOR_H
+
+#include <xapian.h>
+#include <xapian/intrusive_ptr.h>
+#include <xapian/types.h>
+#include <xapian/visibility.h>
+
+#include <xapian-letor/featuremanager.h>
+#include <xapian-letor/featurevector.h>
+#include <xapian-letor/letor.h>
+#include <xapian-letor/letor_features.h>
+#include <xapian-letor/letor_internal.h>
+#include <xapian-letor/ranker.h>
+#include <xapian-letor/ranklist.h>
+
+#include <string>
+#include <map>
+
+#endif /* XAPIAN_INCLUDED_XAPIAN_LETOR_H */
```

Comment:
In github you should be able to see a small pair of red icons, indicating there's no newline at the end of the file. It's better to add one; it makes trailing diffs easier to read in future.